plugins/meta/bandwidth/main.go

```diff
@@ -112,15 +113,16 @@ func getMTU(deviceName string) (int, error) {
 }
 
 func getHostInterface(interfaces []*current.Interface) (*current.Interface, error) {
-	for _, prevIface := range interfaces {
-		if prevIface.Sandbox != "" {
-			continue
+	var err error
+	for _, iface := range interfaces {
+		if iface.Sandbox == "" { // host interface
+			_, _, err = ip.GetVethPeerIfindex(iface.Name)
+			if err == nil {
+				return iface, err
+			}
 		}
-
-		return prevIface, nil
 	}
-
-	return nil, errors.New("no host interface found")
+	return nil, errors.New("no host interface found: " + err.Error())
```

Comment:
if the loop never runs, then `err` won't be set, right?

Reply:
Good catch, that would have nil pointed. Fixed.